stream-chat-android-client/src/main/java/io/getstream/chat/android/client/setup/state/ClientState.kt

```diff
@@ -71,22 +73,20 @@ public interface ClientState {
      */
     public val isInitialized: Boolean
 
+    public val internetAvailable: Boolean
```

Comment:
What do you think about renaming it to `isNetworkAvaiable`?
Also could we add KDOcs?